hw/ip/uart/data/uart.hjson

```diff
@@ -61,10 +61,14 @@
       type:    "status",
       default: "1",
       desc:    "raised if the transmit FIFO is past the high-water mark."}
+    { name:   "tx_empty",
+      type:   "status",
+      default: "1",
+      desc:    "raised if the transmit FIFO has emptied."}
```

Comment:
I think the description should say 'is '...FIFO is empty' otherwise it's likely to lead to confusion and demands for clarification.